packages/@sanity/base/.interopReexport.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * esm/commonjs interop reexport helper
+ * this should be used instead of doing `module.exports = require('./some/file')
+ * since that doesn't work properly with esm imports from the consuming end
+ * @param moduleExports export object from the exporting module
+ * @param importedModule the imported module to be reexported
+ */
+module.exports = function interopReexport(moduleExports, importedModule) {
```

Comment:
Nitpick, but the name of the function could be `_interopReexport` to match the filename.